mirascope/anthropic/utils.py

```diff
@@ -1,10 +1,41 @@
 """A module for utility functions for working with Anthropic."""
 
-from typing import Optional
+from typing import Callable, Optional, Union
 
+from anthropic import Anthropic, AnthropicBedrock, AsyncAnthropic, AsyncAnthropicBedrock
+from anthropic._types import URL
 from anthropic.types import Usage
 
 
+def bedrock_client_wrapper(
+    aws_secret_key: Optional[str] = None,
+    aws_access_key: Optional[str] = None,
+    aws_region: Optional[str] = None,
+    aws_session_token: Optional[str] = None,
+    base_url: Optional[Union[str, URL]] = None,
+) -> Callable[
+    [Union[Anthropic, AsyncAnthropic]], Union[AnthropicBedrock, AsyncAnthropicBedrock]
+]:
+    """Returns a client wrapper for using Anthropic models on AWS Bedrock."""
+
+    def inner_wrapper(client: Union[Anthropic, AsyncAnthropic]):
+        """Returns matching `AnthropicBedrock` or `AsyncAnthropicBedrock` client."""
+        kwargs = {
+            "aws_secret_key": aws_secret_key,
```

Comment:
if user sets in their `.env` file, do they also need to pass it in here? if so, does it make sense for us to get env if it exists?

Reply:
I believe that if set to `None` the underlying provider class will grab it from the environment for us, so no need to duplicate that code.